tests/wpt/mozilla/tests/mozilla/cache_crossorigin_response.sub.html

```diff
@@ -0,0 +1,31 @@
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  let t = async_test("Cached cross-origin response doesn't hang");
+
+  onload = t.step_func(function() {
+      let complete = 0;
+      function check() {
+          complete++;
+          if (complete == 2) {
+              t.done();
+          }
+      }
+
+      const url = "http://{{hosts[][www]]}}:{{ports[http][0]}}/html/semantics/scripting-1/the-script-element/is-script-goal.js?pipe=trickle(d5)|header(Cache-Control,max-age=3600)|header(Pragma,)|header(Expires,36000)";
```

Comment:
When I logged this locally, it came out as `http://www.web-platform.test:8000/html/semantics/scripting-1/the-script-element/is-script-goal.js?pipe=trickle(d5)|header(Cache-Control,max-age=3600)|header(Pragma,)|header(Expires,36000`,
with `location.href` coming out as `http://web-platform.test:8000/_mozilla/mozilla/cache_crossorigin_response.sub.html`. 

So that appears same-origin to me. 

I think changing it to `http://{{hosts[][www1]]}}...` makes it cross-origin.

Reply:
Ah forget about it, I now see the `www.` is missing from the url of the test, so it's basically the same. It might be clearer and more robust to use `www1`, I wasn't aware the test are run without the leading `www`, and that might be subject to change in the future?

Reply:
Clearer is debatable, since there are lots of tests that use the www host. Robust is also debatable, since changing the default host to use a subdomain is not a realistic concern.

Reply:
Makes sense.